README.md

```diff
@@ -79,10 +79,8 @@ This project exists thanks to all the people who contribute.
 <p align="left">
   <a href="https://opencollective.com/the-book-of-secret-knowledge" alt="Financial Contributors on Open Collective">
     <img src="https://img.shields.io/opencollective/backers/the-book-of-secret-knowledge?style=for-the-badge&color=FF4500&labelColor=A9A9A9"></a>
-  </a>
```

Comment:
I would take out the tags at the end of lines 81 & 83 instead but probably not that serious. :)

Reply:
Hi! New to github btw!